src/schema/v2/system/causality_jwt.ts

```diff
@@ -10,6 +10,19 @@ import { ResolverContext } from "types/graphql"
 
 const { HMAC_SECRET } = config
 
+export const causalityJwt = ({ role, userId, saleId, bidderId }) =>
```

Comment:
extracted this method since we are using it outside the resolver now- but not sure where it should go.

Reply:
I considered making a causalityTokenLoader to take care of this but there wasn't an obvious, clean way to do it that would meet the needs of both this file and our general causality `link.ts` file. So for now it is just awkwardly exported here.